avm/res/recovery-services/vault/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -341,15 +341,18 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
     }
     privateEndpoints: [
       {
+        ipConfigurations: {
+          name: 'myIPconfig'
+          properties: {
+            groupId: 'account'
+            memberName: 'default'
+            privateIPAddress: '10.0.0.10'
+          }
+        }
         privateDnsZoneResourceIds: [
           nestedDependencies.outputs.privateDNSZoneResourceId
         ]
         subnetResourceId: nestedDependencies.outputs.subnetResourceId
-        tags: {
```

Comment:
I think for `waf-alignment` you'd need tags. We've seen for other resources that PSRule would start complaining that, as a best practice, you should use (meaningful) tags.

Reply:
Here is the PSRule reference: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.Resource.UseTags

This test does run for this module. If you were to retest it would start 'soft-failing' (i.e., non-blocking) the static tests

Reply:
It spitted out some error when tags where left on 
![Skärmbild 2024-03-12 205534](https://github.com/Azure/bicep-registry-modules/assets/26091353/718af884-25f7-473f-852f-b69605bd98ed)
@AlexanderSehr

Reply:
Hey @alexanderojala, please note that the tags would be expected on the main resource, not the private endpoint (I think). 
That said, if you'd want to add tags to the private endpoint, the UDT would expect you to provide it like this (and not as part of the `ipConfigurations`):
![image](https://github.com/Azure/bicep-registry-modules/assets/5365358/25017e55-face-4b08-8c85-ac9d3ce447c0)

Reply:
![Skärmbild 2024-03-13 205209](https://github.com/Azure/bicep-registry-modules/assets/26091353/e80b25d2-bbbd-4b45-a22b-64accba548d4)
it still give the same error. This is my first module with AVM, so i might missing something. @AlexanderSehr @krbar